rebasehelper/utils.py

```diff
@@ -95,6 +97,36 @@ class ConsoleHelper(object):
 
     """Class for command line interaction with the user."""
 
+    use_colors = False
+
+    @classmethod
+    def should_use_colors(cls, app):
+        """Determine whether ansi colors should be used for CLI output"""
+        if os.environ.get('PY_COLORS') == '1':
+            cls.use_colors = True
+        if os.environ.get('PY_COLORS') == '0':
+            cls.use_colors = False
+        if app.conf.color == 'auto':
+            if (not os.isatty(sys.stdout.fileno()) or
+                    os.environ.get('TERM') == 'dumb'):
+                cls.use_colors = False
+        elif app.conf.color == 'never':
+            cls.use_colors = False
+        cls.use_colors = True
+
+    @classmethod
+    def colored_output(cls, color, string):
```

Comment:
I'd call this method differently, maybe just `print`, set `color` as a keyword argument, by default unset, and probably used string type to avoid importing color functions in other modules. You can use this approach for instance:
```python
import colors

if color and hasattr(colors, color):
    print(getattr(colors, color)(string))
else:
    print(string)
```